variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -77,7 +77,7 @@
      */
     private final Map<String, Set<String>> hgvs;
 
-    public AbstractVariant(String chromosome, int start, int end, String reference, String alternate) {
+    public AbstractVariant(String chromosome, long start, long end, String reference, String alternate) {
```

Comment:
Better than assigning the empty string to the chromosome, I think it should throw an exception if the chromosome argument is null.

Reference and alternate should be okay as null -> empty string, because that is how we represent indels.